src/pages/debrief/cat-c/debrief.cat-c.page.ts

```diff
@@ -94,29 +93,29 @@ export class DebriefCatCPage extends BasePageComponent {
     this.pageState = {
       seriousFaults$: currentTest$.pipe(
         select(getTestData),
-        map(data =>
-          // TODO: MES-4287 Use category C
-          this.faultSummaryProvider.getSeriousFaultsList(data, TestCategory.BE)
+        withLatestFrom(category$),
+        map(([data, category]) =>
+          this.faultSummaryProvider.getSeriousFaultsList(data, category as TestCategory)
```

Comment:
nice forward thinking as we don't know which category would be here due to the sub-categories